code/modules/sanity/sanity_lists.dm

```diff
@@ -169,7 +175,16 @@
 			"The rules are written in blood. The very old blood of generations of idiots.",
 			"You are coming home.",
 			"The writings are here, behind your skull.",
-			"YOU HAVE NO CHOICE"
+			"YOU HAVE NO CHOICE",
+			"Everything is twisted... you... this world... all of it.",
+			"Welcome to the Abyss.",
+			"Human arrogance will eventually usher in our downfall.",
+			"Millions will die. Exciting, don't you think?",
+			"The price we all must pay will be great; atonement shall not be easy. The blame lies with all of us.",
```

Comment:
```suggestion
			"The price we all must pay will be great; atonement shall not be easy. The blame lies on all of us.",
```
"Lie with" is more "to have sexual relations with"

Reply:
Then it should be rests on all of us, no?

iirc the blame lies with you specifically as a reference to sleeping with the blame

Reply:
I was refering myself to the wiktionary
> Used with [with](https://en.wiktionary.org/wiki/with#English): to have sexual relations with.
> _And it came to passe after these things, that his masters wife cast her eyes vpon Ioseph, and shee said, Lie with me._
>
>Used with [on](https://en.wiktionary.org/wiki/on#English)/[upon](https://en.wiktionary.org/wiki/upon#English): to [be](https://en.wiktionary.org/wiki/be) [incumbent](https://en.wiktionary.org/wiki/incumbent) (on); to be the responsibility of a person.

But it's true that on a figurative sense, lie with could work.

Reply:
Yeah you lie with the guilt the same way you lie with the devil, it specifically refers to bedding these concepts